cpp/sorting/Cocktail Sort.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+using namespace std;
+int main() {
+   int arr[] = { 5, 3, 4, 2, 1 };
```

Comment:
Hey @Iampradeepkundra, amazing work. Please make this a user-defined program where the user enters the elements of the array rather than defining it beforehand.  Thankyou :)